src/mame/tomy/kisssite.cpp

```diff
@@ -77,17 +77,19 @@ class kisssite_state : public driver_device
 
 void kisssite_state::machine_reset()
 {
-	m_maincpu->set_state_int(STATE_GENPC, 0xe2cc); // temp, there is code here, but this is unlikely to be the entry point
+	m_maincpu->set_state_int(STATE_GENPC, 0x1cffff80); // might actually be 0x7fffff80 with a ROM mirror
 }
 
 void kisssite_state::mem(address_map &map)
 {
-	map(0x0000000, 0x003ffff).rom();
+	map(0x00000000, 0x0007ffff).ram();
+	map(0x1c000000, 0x1c03ffff).mirror(0x00fc0000).rom();
```

Comment:
You need to add `.region("maincpu", 0)` to make this ROM mapping work. (You also need to raise the address space dimension in the CPU device.)